bigtable-hbase-1.x-replication/pom.xml

```diff
@@ -181,6 +154,11 @@
             <!--       use the HBase server from the HBase environment-->
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud.bigtable</groupId>
+            <artifactId>bigtable-hbase-replication</artifactId>
+            <version>2.0.0-beta6</version>
```

Comment:
we should not hardcode the version, it should be the repo version, use maven tags and speical comment to auto update this version with every release. Refer to other POMs